lib/ApiRequestor.php

```diff
@@ -121,9 +121,9 @@ private function _requestRaw($method, $url, $params, $headers)
 
         if (!$myApiKey) {
             $msg = 'No API key provided.  (HINT: set your API key using '
-              . '"Stripe::setApiKey(<API-KEY>)".  You can generate API keys from '
-              . 'the Stripe web interface.  See https://stripe.com/api for '
-              . 'details, or email support@stripe.com if you have any questions.';
+              . '"Stripe::setApiKey(<API-KEY>)".  You can find your API keys '
```

Comment:
If "Users sometimes surface the error messages to customers" is the problem, mentioning the stripe dashboard is also going to cause confusion.